types/recharts/index.d.ts

```diff
@@ -420,6 +421,14 @@ export interface LegendPayload {
 
 export type BBoxUpdateCallback = (box: { width: number; height: number; }) => void;
 
+export interface LayerProps {
+    className?: string;
+    children?: React.ReactNode[] | React.ReactNode;
+    key?: any; // https://github.com/recharts/recharts/blob/master/demo/component/DemoSankeyLink.js
```

Comment:
I think this is the classic react `key` (like the one you use inside loops - they can be also used outside loops) as I can't find in the code where this prop is defined; plus if it was I think it would be a mistake to have it as it will confuse with the reserved prop from React.

We should drop this line

```suggestion
```

Reply:
agreed